libcloud/test/compute/test_libvirt_driver.py

```diff
@@ -26,11 +28,19 @@ class virConnect:
     A stub/Mock implementation of the libvirt.virConnect class returned by
     the libvirt.openX calles
     """
-    def stub(self, *args, **kwargs):
+    def __stub(self, *args, **kwargs):
         return 0
 
+    def __yes(self, *args, **kwargs):
+        return True
+
+    def __ary(self, *args, **kwargs):
+        return []
+
     def __init__(self):
-        stub = self.stub
```

Comment:
I'm fine with this code, but I believe you could also utilize Mock for that by specifying a class you want to mock for `spec` argument - http://www.voidspace.org.uk/python/mock/mock.html#mock.Mock.

This would mean less code to maintain and keep up to date :)